repos/system_upgrade/el7toel8/actors/checkchrony/libraries/library.py

```diff
@@ -2,6 +2,8 @@
 from leapp.libraries.stdlib import api, run
 
 # Check if the chrony config file was not modified since installation
+
+
```

Comment:
add 1 blank line before the comment instead of 2 between comment and function, so it's clear what the comment refers to